README.md

```diff
@@ -1 +1,50 @@
 # stars-api
```

Comment:
sempre que for instalar uma lib, deve se antes entrar no ambiente virtual venv, isso deve estar na documentação